### PR TITLE
Setup CI + Init project

### PR DESCRIPTION
**Pull request by namtm-1272**
## Related Tickets
- [#51375](https://edu-redmine.sun-asterisk.vn/issues/51375)

## WHAT (optional)
- Init project + setup CI

## HOW
- Init project + setup CI

## WHY (optional)
- Init project + setup CI

### CODE REVIEW

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HnRubyInternFoodAndDrink01</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: Dùng "

**Review by ghost — COMMENTED** [review]
cmt nhỏ

**Comment by namtm-1272** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Comment by namtm-1272** [comment]
ready

**Pull request merged**